lib/statesman/adapters/memory.rb

```diff
@@ -28,7 +28,7 @@ def create(from, to, metadata = {})
         transition
       end
 
-      def last
+      def last(use_cache: true)
```

Comment:
I think that's fine, but I'm not sure about `use_cache`. Would rather have busting the cache as the explicit action, so something like `force_reload` (although that's not perfect).